Objects/dictobject.c

```diff
@@ -1906,6 +1955,16 @@ PyDict_SetItem(PyObject *op, PyObject *key, PyObject *value)
                                  Py_NewRef(key), Py_NewRef(value));
 }
 
+static int
+setitem_lock_held(PyDictObject *mp, PyObject *key, PyObject *value)
+{
+    assert(key);
+    assert(value);
+    return setitem_take2_lock_held(mp,
+                                   Py_NewRef(key), Py_NewRef(value));
```

Comment:
this can fit on one line now

```suggestion
    return setitem_take2_lock_held(mp, Py_NewRef(key), Py_NewRef(value));
```